### PR TITLE
Whitespace/capitalization cleanup in comments and messages.

### PR DESCRIPTION
**Pull request by demiankatz**
As discussed in #2987, this PR removes extra whitespace after periods in comments and a small number of user-facing strings to use a more consistent "single space after period" style. It may not be 100% comprehensive, as I avoided changing the license comment in every single file to keep this manageable. We could do that in a separate PR if there was sufficient desire.

I also fixed a few minor typos (mostly related to capitalization) were noticed while reviewing the whitespace issue, and removed a small number of obsolete/irrelevant comments.

TODO
- [x] Run full test suite before merging

### CODE REVIEW

File: module/VuFindSearch/src/VuFindSearch/Backend/LibGuides/Connector.php [hunk 1]
```diff
@@ -100,7 +100,7 @@ public function __construct($iid, $client, $apiVersion = 1, $baseUrl = null)
     }
 
     /**
-     * Execute a search.  adds all the querystring parameters into
+     * Execute a search. adds all the querystring parameters into
```
Comment (xmorave2) [inline module/VuFindSearch/src/VuFindSearch/Backend/LibGuides/Connector.php]: Could you please make the word 'adds' uppercased?

**Review by xmorave2 — CHANGES_REQUESTED** [review]
Great job @demiankatz! Just one comment (see in code). I think it is worth it to fix it when we are touching it.

File: module/VuFindSearch/src/VuFindSearch/Backend/LibGuides/Connector.php (hunk 1 above)
Comment (demiankatz) [reply module/VuFindSearch/src/VuFindSearch/Backend/LibGuides/Connector.php]: Agreed. I fixed a couple of similar issues but hadn't gotten to this one yet. Fixed now!

File: module/VuFind/src/VuFind/DigitalContent/OverdriveConnector.php [hunk 2]
```diff
@@ -713,7 +713,7 @@ public function lockinResource($overDriveId, $format)
             $this->error("user is not logged in", false, true);
             return $result;
         }
-        // shouldn't need to refresh.  This should be in the cache if it exists
+        // shouldn't need to refresh. This should be in the cache if it exists
```
Comment (xmorave2) [inline module/VuFind/src/VuFind/DigitalContent/OverdriveConnector.php]: On more: shouldn't -> Shouldn't

File: module/VuFind/src/VuFind/ILS/Driver/Innovative.php [hunk 3]
```diff
@@ -404,7 +404,7 @@ public function patronLogin($username, $password)
 
             if (!$api_data['PBARCODE']) {
                 // no barcode found, can look up specific error to return more
-                // useful info.  this check needs to be modified to handle using
+                // useful info. this check needs to be modified to handle using
```
Comment (xmorave2) [inline module/VuFind/src/VuFind/ILS/Driver/Innovative.php]: this -> This

File: module/VuFind/src/VuFind/ILS/Driver/XCNCIP2.php [hunk 4]
```diff
@@ -1978,7 +1978,7 @@ public function getCancelRequestDetails($details)
      *
      * This function returns the item id and recall id as a string
      * separated by a pipe, which is then submitted as form data in Hold.php. This
-     * value is then extracted by the CancelHolds function.  item id is used as the
+     * value is then extracted by the CancelHolds function. item id is used as the
```
Comment (xmorave2) [inline module/VuFind/src/VuFind/ILS/Driver/XCNCIP2.php]: item -> Item

File: module/VuFind/src/VuFind/Search/Base/Options.php [hunk 5]
```diff
@@ -935,7 +935,7 @@ public function showShardCheckboxes()
 
     /**
      * If there is a limit to how many search results a user can access, this
-     * method will return that limit.  If there is no limit, this will return
+     * method will return that limit. If there is no limit, this will return
      * -1.
```
Comment (xmorave2) [inline module/VuFind/src/VuFind/Search/Base/Options.php]: We could maybe move '-1' to the previous line. It would be more readable I think.

**Review by xmorave2 — CHANGES_REQUESTED** [review]
I found some more possible improvements can be done

File: module/VuFind/src/VuFind/DigitalContent/OverdriveConnector.php (hunk 2 above)
Comment (demiankatz) [reply module/VuFind/src/VuFind/DigitalContent/OverdriveConnector.php]: I ended up doing quite a bit more editing of comments in some Overdrive-related files -- a lot of missing capitalization, and some comments that seemed unnecessary or out of context (so I deleted them).

**Comment by demiankatz** [comment]
Okay, I've carefully reviewed all the files myself and run the full integration suite just to be sure nothing has subtly broken. I think this can be merged once you're satisfied, @xmorave2. Thanks again for the help!

**Review by xmorave2 — APPROVED** [review]

**Pull request merged**